packages/synapse-interface/components/_Transaction/components/TimeRemaining.tsx

```diff
@@ -0,0 +1,17 @@
+export const TimeRemaining = ({
+  isComplete,
+  remainingTime,
+  isDelayed,
+}: {
+  isComplete: boolean
+  remainingTime: number
+  isDelayed: boolean
+}) => {
+  if (isComplete) return
+
+  if (isDelayed) {
+    return <div>Waiting...</div>
+  }
+
```

Comment:
In general, there's usage of ms vs seconds vs minutes in the app, and think we should stick to one (prob ms/seconds) and adjust where we need to. Goal is to be consistent so when we add more around all this, there's no confusion about what unit we're dealing with.

Here, for example, `remainingTime` is minutes though some txns will be < than a minute. We can do something like this and use `estTime` on line 16.

```
const estTime = useMemo(() => {
    if (remainingTime > 60) {
      return Math.ceil(remainingTime / 60) + ' minutes'
    } else {
      return remainingTime + ' seconds'
    }
  }, [remainingTime])
```

Reply:
Updated here: https://github.com/synapsecns/sanguine/pull/1892/commits/770029f354ee48eaa6e4ef4fca0670ea5eaa328f